src/main/scripts/package.sh

```diff
@@ -89,96 +98,98 @@ if [[ "$platform" == "linux" ]]; then
     cp "./autogram.template.desktop" "./autogram.desktop"
     sed -i -e "s/PROTOCOL_NAME/$properties_protocol/g" "./autogram.desktop"
 
-    if [[ ! -z "$properties_linux_debMaintainer" ]]; then
+    if [[ ! -z "${properties_linux_appCategory}" ]]; then
         arguments+=(
-            "--linux-deb-maintainer" "$properties_linux_debMaintainer"
+            "--linux-app-category" "${properties_linux_appCategory}"
         )
     fi
 
-    if [[ ! -z "$properties_linux_appCategory" ]]; then
+    if [[ ! -z "${properties_linux_packageDeps}" ]]; then
         arguments+=(
-            "--linux-app-category" "$properties_linux_appCategory"
-        )
-    fi
-
-    if [[ ! -z "$properties_linux_packageDeps" ]]; then
-        arguments+=(
-            "--linux-package-deps" "$properties_linux_packageDeps"
+            "--linux-package-deps" "${properties_linux_packageDeps}"
         )
     fi
 
     lowercase_name=$(echo "$properties_name" | tr '[:upper:]' '[:lower:]')
 
     arguments+=(
-        "--name" "$lowercase_name"
-        "--java-options" "$jvmOptions"
-        "--linux-rpm-license-type" "${properties_linux_rpmLicenseType:-MIT}"
+        "--name" "${lowercase_name}"
+        "--java-options" "${jvmOptions}"
         "--linux-menu-group" "${properties_linux_menuGroup:-Office}"
-        "--install-dir" "$properties_linux_installDir"
+        "--install-dir" "${properties_linux_installDir}"
     )
 
-    if [[ "$properties_linux_shortcut" == "1" ]]; then
+    if [[ "${properties_linux_shortcut}" == "1" ]]; then
         arguments+=(
             "--linux-shortcut"
         )
     fi
 
-    # Build both .rpm and .deb from Linux - should work from Debian-like distro but not the other way around
-    arguments+=(
-        "--type" "rpm"
-    )
-    $jpackage "${arguments[@]}"
-    checkExitCode $?
+    # Load variables of distribution
+    . /etc/os-release
+
+    if [ "${ID}" == "fedora" ]; then
+        arguments+=(
+            "--type" "rpm"
+            "--linux-rpm-license-type" "${properties_linux_rpmLicenseType:-MIT}"
```

Comment:
Tuto nemá byť  fallback na `EUPL-1.2` licenciu? Respektíve to tu ani nemusí byť lebo v defaultoch je LICENSE file https://github.com/slovensko-digital/autogram/pull/369/files#diff-3f30baa0846e0d699e7b883ebd6c765c8a7a9974b23c29e37dc2d378237fe4d7R48

Reply:
@xseman dobry postreh, toto bude tipujem este prezitok z pradavnych cias.